config.json

```diff
@@ -2,87 +2,85 @@
     "azure_mgmt_analysis_services": {
         "azure_mgmt_analysis_services_2016_05_16":{
             "input-file": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/current/specification/analysisservices/resource-manager/Microsoft.AnalysisServices/2016-05-16/analysisservices.json",
-            "namespace": "Azure::ARM::AnalysisServices",
-            "version": "0.11.0",
-            "output-folder": "2016-05-16"
+            "namespace": "Azure::ARM::AnalysisServices::Api_2016_05_16",
+            "package-version": "0.11.0"
```

Comment:
We may be able to generalize `package-version` to one level up in the dictionary so that we only override for the different versions if you like

Reply:
We could, I left it this way to simplify things for now. If we think there's value on globalizing some values, we can add logic to look for these values. Let's discuss this as we put all things together and figure out if it's needed.